### PR TITLE
Create ST Workshop Facilitation Guide.md

### PR DESCRIPTION
**Pull request by shubhi-raft**
### This pull request changes...

Add facilitation guide used to conduct round 4 research workshop with states and territories.

## FUNCTIONALITY
*Describe the functionality*
*Addresses Issue #*

## TO TEST

*List the steps to test the PR*

### This pull request is ready to merge when...

- [X] Meets acceptance criteria for the issue 
- [ ] Code is reviewed by someone other than the original author
- [ ] Code is tested 
- [ ] Experience is approved by UX  
- [ ] Meets [a11y checklist](https://github.com/HHS/TANF-app/blob/main/docs/a11y/how-18f-will-test-a11y.md)
- [ ] Meets [Raft's Manual QASP Checklist](https://teams.microsoft.com/l/file/B5285301-13CA-46FD-A723-AFFD578C224A?tenantId=d58addea-5053-4a80-8499-ba4d944910df&fileType=docx&objectUrl=https%3A%2F%2Fhhsgov.sharepoint.com%2Fsites%2FTANFDataPortalOFA%2FShared%20Documents%2FGeneral%2FManual%20Testing%20Practices_Issue173.docx&baseUrl=https%3A%2F%2Fhhsgov.sharepoint.com%2Fsites%2FTANFDataPortalOFA&serviceName=teams&threadId=19:f769bbcb029f4f02b55ae7fad90e310d@thread.skype&groupId=41f194a6-c1d3-4680-933e-c8ee7d17e287):lock: (only applicable for PR to main HHS)
- [ ] Documentation is updated 
  - OpenAPI  
  - Readme
  - Entity Relationship Diagram (ERD)

### CODE REVIEW

**Comment by shubhi-raft** [comment]
@lfrohlich FYI. adding the guide that existed in hackmd to docs. more research artifacts to follow.

File: docs/User-Research/2020, Winter - User Flow Validation/ST Workshop Facilitation Guide.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Workshop Facilitation Guide for Round 4 Research
+
+We will use the [facilitation guide]((https://app.zenhub.com/files/281707402/8b6beb37-9f66-4526-a6a7-97707f1146ec/download)) to conduct the States and Territories (ST) workshop. The objective of this collaborative design workshop to involve grantees in:
```
Comment (lfrohlich) [inline docs/User-Research/2020, Winter - User Flow Validation/ST Workshop Facilitation Guide.md:3]:
```suggestion
We will use the [facilitation guide]((https://app.zenhub.com/files/281707402/8b6beb37-9f66-4526-a6a7-97707f1146ec/download)) to conduct the States and Territories (ST) workshop. The objective of this collaborative design workshop is to involve grantees in: 
```

File: docs/User-Research/2020, Winter - User Flow Validation/ST Workshop Facilitation Guide.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+- Ice Breaker
+
+#### Solution Space: 10 Minutes  
+In this step, we briefly revisit the problem we are solving and why it is important, for participants who are new to the process. We will realign around the who, what, where, when, and why, and tie it to the TDP product vision.
+
+  - Presentation of solution space
+  - User validation of solution space
+  
+#### Journey Map to User Flow: 45 minutes  
+In this step we will narrate the journey as we currently understand it, ask for ST input on any thing we may have missed, and prioritize pain points.  Then we will present a solution user flow and ask for ST and OFA input .
```
Comment (lfrohlich) [inline docs/User-Research/2020, Winter - User Flow Validation/ST Workshop Facilitation Guide.md:21]:
```suggestion
In this step we will narrate the journey as we currently understand it, ask for ST input on any thing we may have missed, and prioritize pain points.  Then we will present a solution user flow and ask for ST input .  
```

File: docs/User-Research/2020, Winter - User Flow Validation/ST Workshop Facilitation Guide.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# Workshop Facilitation Guide for Round 4 Research
+
+We will use the [facilitation guide]((https://app.zenhub.com/files/281707402/8b6beb37-9f66-4526-a6a7-97707f1146ec/download)) to conduct the States and Territories (ST) workshop. The objective of this collaborative design workshop is to involve grantees in:
```
Comment (lfrohlich) [inline docs/User-Research/2020, Winter - User Flow Validation/ST Workshop Facilitation Guide.md:3]:
```suggestion
We will use the [facilitation guide](https://app.zenhub.com/files/281707402/8b6beb37-9f66-4526-a6a7-97707f1146ec/download) to conduct the States and Territories (ST) workshop. The objective of this collaborative design workshop is to involve grantees in: 
```

**Review by lfrohlich — APPROVED** [review]
Looks good!

**Pull request merged**